manifests/params.pp

```diff
@@ -11,9 +11,12 @@
   $disable_ssl               = false
   $open_ssl_listen_port      = undef
   $postgres_listen_addresses = 'localhost'
-
-  $database                  = 'postgres'
+  
+  $puppetdb_version          = $puppetdb::globals::version
+  $database                  = $puppetdb::globals::database
   $manage_dbserver           = true
+  $manage_pg_repo            = false
```

Comment:
FWIW, this will affect Debian 6 also, but Debian 7 looks like it has a 9.4.

Something more to add, we want to promote the use of PGDG generally so we can promote latest postgres usage, today its 9.4, tomorrow its 9.5/9.6. When 9.6 for example is the one we want users to use, there is something to be said about having our users go in the direction of PGDG, and just be able to use it, without having to go back again and think about manage_repos etc.